Makefile

```diff
@@ -127,6 +127,18 @@ test-go: ## Run tests for backend.
 	@echo "test backend"
 	$(GO) test -v ./pkg/...
 
+test-go-unit: ## Run unit tests for backend.
+	@echo "test backend unit tests"
+	$(GO) test -short -covermode=atomic -timeout=30m ./pkg/...
+
+test-go-integration: ## Run integration tests for backend.
```

Comment:
I would suggest adding a Makefile target for each different backend: sqlite, mysql, postgres by setting appropriately the `GRAFANA_TEST_DB` env variable.
In addition to this, I think that these commands should imitate the way [integrations tests run in the CI (per package)](https://github.com/grafana/grafana/blob/5d74701848e3498d8255a54ff6c5e58594922d6b/scripts/drone/steps/lib.star#L870) so that developers can easier reproduce the failures locally in case the CI tests fail.

Reply:
they could also include starting the test datasource

Reply:
yes, but in that case we need to wait for the containers to be up before starting running the tests.

Reply:
I added a Makefile target for mysql and postgres backends - lmk what you think! I ran the make commands locally and they work